src_assets/common/assets/web/apps.html

```diff
@@ -197,7 +197,8 @@ <h1>Applications</h1>
             </button>
           </div>
           <div class="form-text">
-            A list of commands to be run and forgotten about
+            A list of commands to be run in the background.<br>
+            <b>Note:</b> If the path to the command executable contains spaces, you must enclose it in quotes.
```

Comment:
Just a question... do we accept any standard shell/terminal command including things such as piping, &&, ||, >, etc.

I wonder if we should add an option to select the shell to use and then just pass in their command as an argument to that shell. (Not for this PR, just an idea I've had for a little while)

Reply:
We don't currently, since those are shell builtins rather than real executables. I had similar thoughts when I was working on that URL support for Windows (and we do shell out to cmd.exe in some cases there too).